ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```diff
@@ -69,6 +69,64 @@ tests:
           memory: 100Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-aws
+- as: maistra-istio-unit-3-0
+  commands: |
+    make -e T=-v build racetest binaries-test \
+    BUILD_WITH_CONTAINER="0" \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache" \
+    ARTIFACTS="${ARTIFACT_DIR:-}"
+  container:
+    from: src
+  timeout: 1h0m0s
+- as: integ-pilot-3-0
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  steps:
+    env:
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:3.0
+      MAISTRA_NAMESPACE: maistra-e2e-test
+      MAISTRA_SC_POD: maistra-e2e-test-sc-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        # SRC_PATH does end with /. : the content of the source directory is copied into dest directory
+        oc cp ./. "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":/work/
+      env:
+      - name: MAISTRA_NAMESPACE
+      - name: MAISTRA_SC_POD
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+      timeout: 20m0s
+    - as: integ-pilot-3-0
+      cli: latest
+      commands: |
+        oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
+          entrypoint \
+          prow/integ-suite-kind.sh \
```

Comment:
This failed with:
```
/usr/local/bin/entrypoint: line 79: prow/integ-suite-kind.sh: No such file or directory
```

Do we need to `cd` into the work directory?

cc @yxun

Reply:
No, the copy-src step above copies all source contents into /work/ .

Reply:
@mattmahoneyrh line 127, I think that's a wrong command . You need to remove the "cd /work"

line 127 should be   
  oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}"  \